.github/workflows/assembly.yml

```diff
@@ -40,12 +40,9 @@ jobs:
     - name: Package GeoServer modules and extensions
       run: |
         mvn -B -ntp -nsu -N -f src/pom.xml assembly:single
-    - name: Build community modules without tests
+    - name: Build and package community modules (without tests)
       run: |
-        mvn -B -ntp -U -T3 -DskipTests -PcommunityRelease -f src/community/pom.xml install
-    - name: Package community modules
-      run:  |
-        mvn -B -ntp -nsu -N -f src/community/pom.xml assembly:single
+        mvn -B -ntp -U -T3 -DskipTests -PcommunityRelease,assembly -f src/community/pom.xml install
```

Comment:
Similar change needed for `build/publish_release.sh`

Reply:
Checked, it actually does not need it, as community modules are not assembled for a release.
However, the nightly build jenkins scripts will have to be updated on merge.